src/main/java/de/fraunhofer/aisec/cpg/graph/edge/PropertyEdge.java

```diff
@@ -19,7 +19,7 @@
   /** Required field for object graph mapping. It contains the node id. */
   @Id @GeneratedValue private Long id;
 
-  protected static final Logger log = LoggerFactory.getLogger(PropertyEdge.class);
+  @Transient protected static final Logger log = LoggerFactory.getLogger(PropertyEdge.class);
```

Comment:
Wouldn't it be better to just skip static fields in the Codyze - similarly to what we already do for Nodes? It is very error-prone to have every static field annotated with `@Transient`

Reply:
I agree, however @Transient should also be supported.